packages/openapi3/test/info.test.ts

```diff
@@ -94,4 +94,17 @@ describe("openapi3: info", () => {
       },
     });
   });
+
+  it("set the service description with /** */ Doc comments", async () => {
```

Comment:
Doc comment are just a syntax sugar for doc decorator we don’t need to test both in libraries